templates/details.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html'%} {% block content %}
+<div class="container">
+  <div>
+    <img
+      class="img-thumbnail"
+      src="{{user.img_url}}"
+      style="display: inline"
+      width="180"
+      height="200"
+    />
+    <h1 style="display: inline">{{user.first_name}} {{user.last_name}}</h1>
+  </div>
+
+  <div class="row mx-3 mt-3">
+    <form action="{{user.id}}/edit">
```

Comment:
Just in case: remember to also update here once you change the route names in `app.py`.